package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mongtype",
-  "version": "4.1.0",
+  "version": "4.2.0",
```

Comment:
With the addition of the `RepoOperation` enum, this might be a "breaking change". I think a major version bump would be in order

Reply:
wait, why would this be breaking? It adds an interface and an optional param which uses it. It should not break existing code at all.